deploy/cr.yaml

```diff
@@ -86,6 +86,62 @@ spec:
           requests:
             storage: 6Gi
     gracePeriod: 600
+  haproxy:
+    enabled: false
```

Comment:
I believe `haproxy` should be enabled by default instead of `proxysql`

Reply:
And we need to add/fix tests.

Reply:
tests are ok for separate PR